packages/models-library/src/models_library/aiodocker_api.py

```diff
@@ -11,7 +11,7 @@
 
 
 class AioDockerContainerSpec(ContainerSpec):
-    Env: dict[str, str | None] | None = Field(
+    Env: dict[str, str | None] | None = Field(  # type: ignore
```

Comment:
why this ignore? try `Field(default=None, description="some description")` !

Reply:
`Incompatible types in assignment (expression has type "Optional[Dict[str, Optional[str]]]", base class "ContainerSpec" defined the type as "Optional[List[str]]")  [assignment]`  I guess thats the reason why we have `convert_list_to_dict` validator